modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+2        false    -         -       -           -         -             -
+3        false    -         -       -           -         -             -
+```
+
+[[decom-state]]
+== Brokers in a decommissioned state
+
+Ensure that no brokers are in a decommissioned state.
+
+Example:
+```bash 
+rpk redpanda admin brokers list
+```
+
+Output:
+```bash
+NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
+0        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+1        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+2        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+```
+
+
+== Level 2 Production Readiness
+
+At level 2 we are looking to ensure that you are best able to monitor and support your environment on a sustained basis. The following is a list of recommendations to help this.
+
+. Do you have a development or test environment configured where you can evaluate upgrades and new versions before rolling them straight to production.
+
+. Monitoring: Prometheus and Grafana, and/or Datadog or equivalent are configured to scrape metrics from all Redpanda brokers at a regular interval.
+
+. System logs are being captured and stored for an appropriate period of time (usually at least 7 days). On bare metal, this may be journald however on Kubernetes you may need to have fluentd or equivalent configured and logs sent to a central location.
+
+. Upgrade policy defined and implemented - Redpanda Enterprise supports rolling upgrades, so upgrades won’t need downtime, however make sure that upgrades are scheduled on a regular basis, ideally via automation such as Ansible or Helm.
+
+. For higher availability requirements, is the cluster configured across multiple availability zones / fault domains and is rack_awareness configured correctly?
+
++
+```bash
+rpk cluster info
+```
++
+```bash
+CLUSTER
+=======
+redpanda.be267958-279d-49cd-ae86-98fc7ed2de48
+
+BROKERS
+=======
+ID    HOST            PORT  RACK
+0*    54.70.51.189    9092  us-west-2a
+1     35.93.178.18    9092  us-west-2b
+2     35.91.121.126   9092  us-west-2c
+```
++
+```bash
+rpk cluster config get enable_rack_awareness
+```
++
+```bash
+true
+```
+
+== Level 3 Production Readiness
+
+At level 3 we are looking at full enterprise readiness and the ability to operate at the highest levels of availability and the ability to prevent and recover from the most serious of incidents.
+
+https://docs.redpanda.com/current/reference/monitor-metrics/
+. Alerts configured for metrics. A standard set of alerts for Grafana or Prometheus is provided, however should be customized for your specific needs. Are alerts configured to proactively alert operators if there is an issue? 
+
+. Is a backup or disaster recovery (DR)  solution in place? Backup and restore can be configured via Tiered Storage using the Whole Cluster Recovery feature.
```

Comment:
```suggestion
- Confirm that a backup and disaster recovery (DR)  solution in place. You can configure backup and restore using the Tiered Storage Whole Cluster Recovery feature.
```

Reply:
Be sure to make Tiered Storage Whole Cluster Recovery links to https://docs.redpanda.com/current/manage/whole-cluster-restore/

Reply:
See also:
- https://docs.redpanda.com/current/manage/tiered-storage/